### PR TITLE
Move a couple of Akka Streams examples to their own files

### PR DESCRIPTION
**Pull request by raboof**
Might make it easier to follow?

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
LGTM

File: akka-docs/src/main/paradox/stream/operators/Source-or-Flow/log.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This can be changed by calling @scala[`Attributes.logLevels(...)`] @java[`Attrib
 ## Example
 
 Scala
-:   @@snip [SourceOrFlow.scala](/akka-docs/src/test/scala/docs/stream/operators/SourceOrFlow.scala) { #log }
+:   @@snip [SourceOrFlow.scala](/akka-docs/src/test/scala/docs/stream/operators/sourceorflow/Log.scala) { #log }
 
 Java
 :   @@snip [SourceOrFlow.java](/akka-docs/src/test/java/jdocs/stream/operators/SourceOrFlow.java) { #log }
```
Comment (patriknw) [inline akka-docs/src/main/paradox/stream/operators/Source-or-Flow/log.md]: fix .java at the same time?

**Comment by patriknw** [comment]
PLS BUILD

**Review by patriknw — APPROVED** [review]
LGTM, can be merged after additional validation build

**Pull request merged**